design-tokens/tokens/component.default.json

```diff
@@ -8112,6 +8112,18 @@
             }
           }
         }
+      },
+      "gapX": {
+        "$type": "number",
+        "$value": "{component.small.textToElementX}",
```

Comment:
I think this should use `textToIconX`?

Reply:
Can we update these to that for now? There's a separate ticket open to revisit naming/usage of `textToElement` vs `textToIcon`.

Reply:
aw okay I wondered but I didnt think it was `textToIcon` because that's `6px` and we have `12px`

Reply:
are we okay with it being `6px`?

<img width="651" alt="Screen Shot 2024-08-07 at 7 18 47 PM" src="https://github.com/user-attachments/assets/856d3e16-8c57-48ae-a895-5927cbf74f35">

Reply:
Ah thanks for clarifying. Yes, let's do it at `6px` rn and it brings up a good discussion of why some components have 6px gap while others have 12px.

Reply:
sounds good will do!